sites/platform/src/create-apps/multi-app/project-structure.md

```diff
@@ -55,7 +55,7 @@ They all have different configurations for how they serve the files. For more de
 
 {{< note >}}
 
-The `.platform` directory is located at the root, separate from your apps.
+The `{{% vendor/configdir %}}` directory is located at the root, separate from your apps.
```

Comment:
This subbed out correctly - I think its just the parameter I didn't set back while testing that's the issue here.